File: JAVA/70. Climbing Stairs.java
class Solution {
    public int climbStairs(int n) {
        if(n==1)
        return 1;
        if(n==2)
        return 2;
        int a=1,b=2,c=0;
        for(int i=2;i<=n;i++){
            c=a+b;
            a=b;
            b=c;
        }
        return a;
    }
}

File: C++/70. Climbing Stairs.cpp
class Solution {
public:
    int climbStairs(int n) {
        if(n<4)
        return n;
        int a=2,b=3;
        for(int i=3;i<n;i++){
            int t=a+b;
            a=b;
            b=t;
        }
        return b;
    }
};